Dockerfile

```diff
@@ -1,53 +1,66 @@
-ARG GOLANG_BASE
-ARG DEBIAN_BASE
-
-FROM ${GOLANG_BASE} as build
+FROM docker.io/golang:1.18.1-bullseye@sha256:3b1a72af045ad0fff9fe8e00736baae76d70ff51325ac5bb814fe4754044b972 AS build
```

Comment:
This base image is different from the one that we used in the script.
I assume you haven't commented on the data as we did before. We want to let dependabot or renovate to upgrade this regularly.

No action here. I just want to put a comment for future reader.